pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test: TestLogic_crdb_internal failed

pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test.TestLogic_crdb_internal [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10543449?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10543449?buildTab=artifacts#/) on release-22.2 @ [96933e7f333ceb2e33df7758264c24743d4a8caa](https://github.com/cockroachdb/cockroach/commits/96933e7f333ceb2e33df7758264c24743d4a8caa):


```
=== RUN   TestLogic_crdb_internal
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/f0e190bbb53e1cb64e2af579206a2d9d/logTestLogic_crdb_internal2788510474
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
[11:40:58] rng seed: 297034990968416539

    logic.go:2775: let $testdb_id = 106
    logic.go:2775: let $testdb_foo_id = 108
    logic.go:2775: let $schema_bar_id = 111
=== CONT  TestLogic_crdb_internal
    logic.go:3940: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/f0e190bbb53e1cb64e2af579206a2d9d/logTestLogic_crdb_internal2788510474
--- FAIL: TestLogic_crdb_internal (48.42s)
=== RUN   TestLogic_crdb_internal/max_retry_counter
    logic.go:2718: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2984/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/crdb_internal:852: SELECT start_pretty, end_pretty FROM crdb_internal.ranges
        WHERE split_enforced_until IS NOT NULL
        AND (start_pretty LIKE '/Table/112/1%' OR start_pretty LIKE '/Table/112/2%')
        expected:
            /Table/112/1/1  /Table/112/1/2
            /Table/112/1/2  /Table/112/1/3
            /Table/112/1/3  /Table/112/2/1
            /Table/112/2/1  /Table/112/2/2
            /Table/112/2/2  /Table/112/2/3
            /Table/112/2/3  /Table/112/3/1
            
        but found (query options: "retry") :
[11:41:45] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2984/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/crdb_internal: 118 statements
    logic.go:2032: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2984/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/crdb_internal:864: too many errors encountered, skipping the rest of the input
[11:41:45] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2984/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/crdb_internal with config local-legacy-schema-changer: 118 tests, 2 failures
[11:41:46] --- total progress: 118 statements
--- total: 118 tests, 2 failures
    --- FAIL: TestLogic_crdb_internal/max_retry_counter (46.03s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_crdb_internal.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-28789

pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test.TestLogic_crdb_internal [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10575248?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10575248?buildTab=artifacts#/) on release-22.2 @ [8055184fc991424d1d0eefd7ccf703948b8de3ee](https://github.com/cockroachdb/cockroach/commits/8055184fc991424d1d0eefd7ccf703948b8de3ee):


```
=== RUN   TestLogic_crdb_internal
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/f0e190bbb53e1cb64e2af579206a2d9d/logTestLogic_crdb_internal2124701408
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
[11:30:00] rng seed: -6960161890907838940

    logic.go:2775: let $testdb_id = 106
    logic.go:2775: let $testdb_foo_id = 108
    logic.go:2775: let $schema_bar_id = 111
=== CONT  TestLogic_crdb_internal
    logic.go:3940: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/f0e190bbb53e1cb64e2af579206a2d9d/logTestLogic_crdb_internal2124701408
--- FAIL: TestLogic_crdb_internal (47.83s)
=== RUN   TestLogic_crdb_internal/max_retry_counter
    logic.go:2718: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2985/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/crdb_internal:852: SELECT start_pretty, end_pretty FROM crdb_internal.ranges
        WHERE split_enforced_until IS NOT NULL
        AND (start_pretty LIKE '/Table/112/1%' OR start_pretty LIKE '/Table/112/2%')
        expected:
            /Table/112/1/1  /Table/112/1/2
            /Table/112/1/2  /Table/112/1/3
            /Table/112/1/3  /Table/112/2/1
            /Table/112/2/1  /Table/112/2/2
            /Table/112/2/2  /Table/112/2/3
            /Table/112/2/3  /Table/112/3/1
            
        but found (query options: "retry") :
[11:30:47] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2985/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/crdb_internal: 118 statements
    logic.go:2032: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2985/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/crdb_internal:864: too many errors encountered, skipping the rest of the input
[11:30:47] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2985/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/crdb_internal with config local-legacy-schema-changer: 118 tests, 2 failures
[11:30:47] --- total progress: 118 statements
--- total: 118 tests, 2 failures
    --- FAIL: TestLogic_crdb_internal/max_retry_counter (45.93s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_crdb_internal.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test.TestLogic_crdb_internal [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10582274?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10582274?buildTab=artifacts#/) on release-22.2 @ [8055184fc991424d1d0eefd7ccf703948b8de3ee](https://github.com/cockroachdb/cockroach/commits/8055184fc991424d1d0eefd7ccf703948b8de3ee):


```
=== RUN   TestLogic_crdb_internal
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/f0e190bbb53e1cb64e2af579206a2d9d/logTestLogic_crdb_internal3902424955
    test_log_scope.go:79: use -show-logs to present logs inline
*
* INFO: Running test with the default test tenant. If you are only seeing a test case failure when this message appears, there may be a problem with your test case running within tenants.
*
[11:37:35] rng seed: 3604903900280120894

    logic.go:2775: let $testdb_id = 106
    logic.go:2775: let $testdb_foo_id = 108
    logic.go:2775: let $schema_bar_id = 111
=== CONT  TestLogic_crdb_internal
    logic.go:3940: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/f0e190bbb53e1cb64e2af579206a2d9d/logTestLogic_crdb_internal3902424955
--- FAIL: TestLogic_crdb_internal (47.72s)
=== RUN   TestLogic_crdb_internal/max_retry_counter
    logic.go:2718: 
         
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3009/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/crdb_internal:852: SELECT start_pretty, end_pretty FROM crdb_internal.ranges
        WHERE split_enforced_until IS NOT NULL
        AND (start_pretty LIKE '/Table/112/1%' OR start_pretty LIKE '/Table/112/2%')
        expected:
            /Table/112/1/1  /Table/112/1/2
            /Table/112/1/2  /Table/112/1/3
            /Table/112/1/3  /Table/112/2/1
            /Table/112/2/1  /Table/112/2/2
            /Table/112/2/2  /Table/112/2/3
            /Table/112/2/3  /Table/112/3/1
            
        but found (query options: "retry") :
[11:38:22] --- progress: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3009/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/crdb_internal: 118 statements
    logic.go:2032: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3009/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/crdb_internal:864: too many errors encountered, skipping the rest of the input
[11:38:22] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3009/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/local-legacy-schema-changer/local-legacy-schema-changer_test_/local-legacy-schema-changer_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/crdb_internal with config local-legacy-schema-changer: 118 tests, 2 failures
[11:38:22] --- total progress: 118 statements
--- total: 118 tests, 2 failures
    --- FAIL: TestLogic_crdb_internal/max_retry_counter (45.89s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_crdb_internal.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/logictest/testdata/logic_test/crdb_internal
+++ b/pkg/sql/logictest/testdata/logic_test/crdb_internal
@@ -853,12 +853,6 @@ SELECT start_pretty, end_pretty FROM crdb_internal.ranges
 WHERE split_enforced_until IS NOT NULL
 AND (start_pretty LIKE '/Table/112/1%' OR start_pretty LIKE '/Table/112/2%')
 ----
-/Table/112/1/1  /Table/112/1/2
-/Table/112/1/2  /Table/112/1/3
-/Table/112/1/3  /Table/112/2/1
-/Table/112/2/1  /Table/112/2/2
-/Table/112/2/2  /Table/112/2/3
-/Table/112/2/3  /Table/112/3/1
 
 statement ok
 ALTER TABLE foo SPLIT AT VALUES (1), (2), (3)

